app/modules/ads/index.js

```diff
@@ -357,6 +358,21 @@ class Ads {
     }
   }
 
+  /**
+   * @private
+   */
+  trackConnectionDownlinkToDW() {
+    const { utils } = window.Wikia.adEngine;
+    const connection = navigator.connection
+      || navigator.mozConnection
+      || navigator.webkitConnection;
+
+    if (connection) {
+      pageTracker.trackProp('downlink', connection.downlink);
```

Comment:
What about naming it `bandwidth`?

Reply:
I would keep the name of the actual property. Moreover because it is not the same as bandwidth: https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation/downlink

Reply:
> The downlink read-only property of the NetworkInformation interface returns the effective bandwidth estimate in megabits per second